wfe/wfe.go

```diff
@@ -1244,12 +1244,6 @@ func (wfe *WebFrontEndImpl) Authorization(ctx context.Context, logEvent *web.Req
 	logEvent.Extra["Identifier"] = authz.Identifier
 	logEvent.Extra["AuthorizationStatus"] = authz.Status
 
-	// After expiring, authorizations are inaccessible
```

Comment:
Given we plan to turn this back on could you just comment it out with a `TODO` and open a issue to turn the behavior back on.

Reply:
I don't like to leave commented out code checked in, but I've filed an issue to restore the 404s: https://github.com/letsencrypt/boulder/issues/3781